src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -2138,6 +2144,17 @@ void MainWindow::slot_fullscreen()
         button->set_fullscreen(m_fullscreen);
 }
 
+void MainWindow::slot_check_fullscreen()
+{
+    QAction* fullscreen = m_ui->menu_view->findChild<QAction*>("Fullscreen");
```

Comment:
It would be cleaner IMHO to store the action in the class and use it, instead of looking for it by name every time.

Reply:
Good idea!